### DISCUSSION TITLE
Script to apply default or custom configuration

### DISCUSSION THREAD

**Discussion by danielniccoli** [Dadido3/noita-mapcap#12] [Ideas]
Hi David,

I prepared a script that does reset the game configuration and apply recommended settings, as well as customised settings.

I was wondering if it would be possible to read the `CAPTURE_GRID_SIZE` from a json file? I'm not so much into Noita modding and LUA, which is why I ask. But I will provide the script, once done.

**Comment by Dadido3** [comment]
The JSON lib i added only supports writing (for now), you would need to use another JSON lib, there are plenty to choose from.

The better way would be to use the built in UI stuff for mod settings, this way a user can configure everything from the settings menu. As example, see:
![grafik](https://user-images.githubusercontent.com/6549004/179578420-5a0831c5-d85f-4463-bb4d-49af1d3bc990.png)

I would need to check how it is done. I guess `...\Nolla_Games_Noita\data\scripts\lib\mod_settings.lua` is the place to start investigating.
